lock-api/src/main/java/com/palantir/lock/LockServerOptions.java

```diff
@@ -86,7 +86,10 @@ public TimeDuration getMaxNormalLockAge() {
     /**
      * Returns the number of bits used to create random lock token IDs. The
      * default value is 64 bits.
+     *
+     * @deprecated this value is no longer used or respected
```

Comment:
started an internal document with things we should call out for the next release